spec/support/helpers/ask_export_helper.rb

```diff
@@ -22,71 +22,59 @@ def smart_survey_response(items)
   def smart_survey_row(options = {})
     environment = options.fetch(:environment, :draft)
     config = AskExport::CONFIG[environment]
+
+    to_merge = {}
```

Comment:
Minor: can we define the return hash here, and then update it with this? I'd find that easier, as it's one less concept to keep in mind.